File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int> MinHeap;
        int s=nums.size();
        int ans;
        for(auto i=0;i<s;i++){
            MinHeap.push(-1*nums[i]);
            if(MinHeap.size()>k){
                MinHeap.pop();
            }
        }
        ans=-1*MinHeap.top();
        return ans;
    }
};

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq=new PriorityQueue<>();
        //Scanner sc=new Scanner(System.in);
        for(int i=0;i<nums.length;i++){
            pq.add(-1*nums[i]);
        }
        int ans=0;
        for(int i=0;i<k;i++){
            ans=pq.remove();
            
        }
        return -1*ans;
    }
}